graphql-dgs-example-java/build.gradle.kts

```diff
@@ -19,4 +19,6 @@ dependencies {
     implementation(project(":graphql-dgs-subscriptions-websockets-autoconfigure"))
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("io.projectreactor:reactor-core")
+    implementation(project(":graphql-dgs-spring-boot-micrometer"))
```

Comment:
you will not this anymore as part of the pr. I just merged https://github.com/Netflix/dgs-framework/pull/297.

Reply:
This PR removed micrometer from the example project.

Reply:
Sound good! Thanks, that would be helpful!